### PR TITLE
fix logic in method_missing

### LINKED ISSUE #754: rake task now failing
Hello,

Sorry i dont have much more information but since the last update to 2.3.1 my rake tasks are failing on linux machines.  this can be seen in the travis task hear

https://travis-ci.org/icann-dns/puppet-knot/jobs/280395205

the check `bundle exec rake test` fails on the on the lint task from puppet-lint with an exit code of 1.  changing back to puppet-lint 2.3.0 fixes the problem as can be seen on the following travis-ci job.  

https://travis-ci.org/icann-dns/puppet-knot/jobs/280415804

happy to to more testing but not really sure where to start

### PR DESCRIPTION
**Pull request by tphoney**
This should fix #754

### CODE REVIEW

File: lib/puppet-lint/configuration.rb [hunk 1]
```diff
@@ -54,7 +54,7 @@ def method_missing(method, *args, &_block)
 
       option = Regexp.last_match(:option)
       add_option(option.to_s) if settings[option].nil?
-      settings[option] = args[0] if args.length > 1
+      settings[option] = args[0] if args.length > 0
```
Comment (rnelson0) [inline lib/puppet-lint/configuration.rb]: Rubocop says `Use !empty? instead of length > 0`, if you can adjust this.

**Comment by alexjfisher** [comment]
LGTM.  Regression was in https://github.com/rodjek/puppet-lint/commit/6fa8111e1692630304dc8f17824997e9410425e1

@tphoney Rubocop fails with.
> lib/puppet-lint/configuration.rb:57:37: C: Style/ZeroLengthPredicate: Use !empty? instead of length > 0

**Comment by tphoney** [comment]
also @DavidS 👍

**Comment by tphoney** [comment]
@rnelson0 should be good now

**Comment by tphoney** [comment]
@rnelson0 are you going to push a release ?

**Comment by rnelson0** [comment]
Need a release PR. I'm trying to get clear of issues at work, so I can get it after lunch, or if you can model one after #753 I can click merge and then do the release.

**Comment by tphoney** [comment]
#758  is there

**Comment by rnelson0** [comment]
2.3.2 is published, please ensure it works for you.

**Comment by tphoney** [comment]
works for me @rnelson0, thanks for all the help

**Pull request merged**